src/test/java/org/jenkinsci/plugins/workflow/log/TaskListenerDecoratorOrderTest.java

```diff
@@ -0,0 +1,189 @@
+package org.jenkinsci.plugins.workflow.log;
+
+import com.google.common.collect.ImmutableSet;
+import hudson.console.ConsoleLogFilter;
+import hudson.console.LineTransformationOutputStream;
+import hudson.model.AbstractBuild;
+import hudson.model.Node;
+import hudson.model.TaskListener;
+import hudson.remoting.Channel;
+import jenkins.security.MasterToSlaveCallable;
+import jenkins.util.JenkinsJVM;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.flow.FlowExecutionOwner;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.jenkinsci.plugins.workflow.steps.*;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.*;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.Set;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.lessThan;
+
+@Issue("SECURITY-2128")
```

Comment:
```suggestion
```

Misleading; this is not correcting a security vulnerability.

Reply:
removed and updated PR